stubs/jsonschema/jsonschema/_format.pyi

```diff
@@ -1,15 +1,16 @@
 from collections.abc import Callable, Iterable
-from typing import Any, TypeVar
+from typing import Any, TypeVar, Union
 
 _F = TypeVar("_F", bound=Callable[..., Any])
+_RaisesType = Union[type[Exception], tuple[type[Exception], ...]]
```

Comment:
```suggestion
from typing import Any, TypeAlias, TypeVar, Union

_F = TypeVar("_F", bound=Callable[..., Any])
_RaisesType: TypeAlias = Union[type[Exception], tuple[type[Exception], ...]]
```

Reply:
Thanks! I was just now trying to figure out how I'm supposed to write this. I'm still not accustomed to using `TypeAlias` yet.

Reply:
Need to import `TypeAlias` from `typing_extensions`; `typing.TypeAlias` is only a thing on 3.10+